modules/ts/src/ts_perf.cpp

```diff
@@ -816,24 +816,24 @@ void TestBase::Init(const std::vector<std::string> & availableImpls,
     param_seed          = args.get<unsigned int>("perf_seed");
     param_time_limit    = std::max(0., args.get<double>("perf_time_limit"));
     param_force_samples = args.get<unsigned int>("perf_force_samples");
-    param_write_sanity  = args.has("perf_write_sanity");
-    param_verify_sanity = args.has("perf_verify_sanity");
+    param_write_sanity  = args.get<bool>("perf_write_sanity");
+    param_verify_sanity = args.get<bool>("perf_verify_sanity");
 
 #ifndef WINRT
-    test_ipp_check      = !args.has("perf_ipp_check") ? getenv("OPENCV_IPP_CHECK") != NULL : true;
+    test_ipp_check      = !args.get<bool>("perf_ipp_check") ? getenv("OPENCV_IPP_CHECK") != NULL : true;
 #else
     test_ipp_check = false;
 #endif
     param_threads       = args.get<int>("perf_threads");
 #ifdef CV_COLLECT_IMPL_DATA
-    param_collect_impl  = args.has("perf_collect_impl");
+    param_collect_impl  = args.get<bool>("perf_collect_impl");
 #endif
 #ifdef ANDROID
     param_affinity_mask   = args.get<int>("perf_affinity_mask");
     log_power_checkpoints = args.has("perf_log_power_checkpoints");
```

Comment:
I can do that for consistency. However it is not necessary because there is no default value for this one.

The conditions are:
1. default "false" + get<bool>()
2. no default value + has()